lib/llm_composer/models/ollama.ex

```diff
@@ -77,7 +78,12 @@ defmodule LlmComposer.Models.Ollama do
 
       %Message{type: :assistant, content: message} ->
         %{"role" => "assistant", "content" => message}
+
+      # system prompt nil case
+      nil ->
+        nil
```

Comment:
```suggestion
      %Message{type: :system, content: message} when message in [nil, ""]->
        nil
```

Instead of making it nil before it gets here and putting the comment to explain, why don't we just pattern match here? This match arm would need to be before the other `:system` one

Reply:
Thanks for the suggestion!